metadata/src/test/java/org/apache/kafka/controller/QuorumControllerTest.java

```diff
@@ -350,6 +350,7 @@ public void testFenceMultipleBrokers() throws Throwable {
         }
     }
 
+    @Disabled
```

Comment:
I think we forgot a `@Test` here.

Reply:
> I think we forgot a `@Test` here.
thanks your reply.
I saw that there was a “@Test” annotation before, but it was removed at this [commit](https://github.com/apache/kafka/commit/8c88cdb7186b1d594f991eb324356dcfcabdf18a). I wonder if it was intentional? @dengziming

Reply:
I guess it was removed accidentally since there are no related discussion about this change.

Reply:
> I guess it was removed accidentally since there are no related discussion about this change.

Ok thanks, I've fixed it. @dengziming